content/blog/2025/09/kubewarden-1.29-release.md

```diff
@@ -0,0 +1,178 @@
+checked so it doesn't exceed `spec.timeoutSeconds`, and so neither
+`timeoutSeconds` nor `timeoutEvalSeconds` can be greater than the maximum
+webhook timeout from Kubernetes (hardcoded at 30s).
+
+In the case of AdmissionPolicies and ClusterAdmissionPolicies, this new field
+is part of the [usual spec](https://docs.kubewarden.io/reference/CRDs#policyspec).
+
+For AdmissionPolicyGroups and ClusterAdmissionPolicyGroups, the new field is
+[part of each
+member](https://docs.kubewarden.io/reference/CRDs#policygroupmember) instead of
+the global policy group spec, which allows for more fine tuning.
+
+## container-resources policy improvements
+
+This policy enforces memory and CPU requests and limits. Since version
+`v1.1.0`, it now supports `minRequest` validation for both CPU and memory:
+
+```yaml
+memory:
+  defaultRequest: "100M"
+  defaultLimit: "500M"
+  minRequest: "50M" # new
+  maxLimit: "4G"
+cpu:
+  defaultRequest: 100m
+  defaultLimit: 200m
+  minRequest: 50m # new
+  maxLimit: 500m
+```
+
+As with the pre-existing values, the policy verifies that the `defaultRequest`
+must be between `minRequest` and `maxLimit`, and that the `defaultLimit` must
+be less than `maxLimit`.
+
+## persistentvolumeclaim-storageclass policy improvements
+
+This policy checks the use of StorageClasses with PersistentVolumeClaims.
+Since version `v1.1.0`, it now supports an allow-list in the form of:
+
+```yaml
+# A list of storage classes that are permitted.
+# Mutually exclusive with deniedStorageClasses.
+allowedStorageClasses: # new
+  - standard
+  - slow
+```
+
+This new `allowedStorageClasses` setting cannot be used in conjunction with
+`deniedStorageClasses` (you either have an allow-list, or a deny-list). One
+can always deploy several policies, though.
+
+## ContainerDays Hamburg 2025
+
+One of our maintainers (Víctor, your narrator today!) had a speaker slot
+on [ContainerDays 2025
+Hamburg](https://www.containerdays.io/containerdays-conference-2025), with the
+talk _"Simplifying cluster security with CEL"_. In it we gave an overview on
+the CEL language, its uses in both Kubernetes Admission Controllers and Dynamic
+Admission Controllers such as Kubewarden, and explained how we expanded on it
+for Kubewarden.
+
+Stay tuned to their [YouTube channel](https://www.youtube.com/@ContainerDays)
+to watch the recording!
+
+## Google Summer of Code wrapped up!
+
+We are happy to congratulate Esosa Ohangbon, our Google Summer Of Code
+mentee, on successfully completing the mentorship! 🎉🥳🎊
+
+Esosa worked on fleshing out a new [Policy SDK for
+Typescript/Javascript](https://www.npmjs.com/package/kubewarden-policy-sdk).
+The end result is a tech-preview of a useful SDK. We look forward to
+collaborations such as this one!
+
+## Signing Hauler manifest
+
+Since Kubewarden 1.28, we also ship a [Hauler
+manifest](../08/kubewarden-1.28-release/) together with our Helm charts to
+simplify air-gap installs.
```

Comment:
```suggestion
Since [Kubewarden 1.28](../08/kubewarden-1.28-release/), we also ship a [Hauler
manifest](https://docs.hauler.dev/docs/intro) together with our Helm charts to
simplify air-gap installs.
```